mklove/modules/configure.libcurl

```diff
@@ -45,8 +45,8 @@ void foo (void) {
 function install_source {
     local name=$1
     local destdir=$2
-    local ver=7.86.0
-    local checksum="3dfdd39ba95e18847965cd3051ea6d22586609d9011d91df7bc5521288987a82"
+    local ver=8.7.1
```

Comment:
Adding a note for the team here that curl 8 has no API or ABI changes

Reply:
@milindl 
Thank you. Can you please see this commit through completion - this is as far as I can go with it.